library/std/src/sys/windows/compat.rs

```diff
@@ -262,7 +262,7 @@ macro_rules! compat_fn_optional {
             #[allow(unused)]
             pub(in crate::sys) fn preload(module: Module) {
                 unsafe {
-                    let symbol_name = ansi_str!(sym $symbol);
+                    static symbol_name: &CStr = ansi_str!(sym $symbol);
```

Comment:
Shouldn't statics be named in UPPER_CASE?

Reply:
Oh hm. And the checker can't see in macros. One sec...